augur/db/export/sqlite3.py

```diff
@@ -0,0 +1,42 @@
+from augur.io.sqlite3 import Sqlite3Database, sanitize_identifier
+from ..defaults import DEFAULT_SEQUENCES_ID_NAME, DEFAULT_SEQUENCES_SEQ_NAME
+
+
+def export(db_file:str,
+        metadata_table:str, sequences_table:str,
+        output_metadata:str=None, output_sequences:str=None):
+    """Export data from a SQLite3 database file."""
+    database = Sqlite3Database(db_file)
+
+    if output_metadata:
+        export_metadata(database, metadata_table, output_metadata)
+
+    if output_sequences:
+        export_sequences(database, sequences_table, output_sequences)
+
+
+def export_metadata(database:Sqlite3Database, table_name:str, output_file:str):
+    """Export metadata from a SQLite3 database."""
+
+    query = f"SELECT * FROM {sanitize_identifier(table_name)}"
+
+    database.query_to_csv(
+        query=query,
+        path=output_file,
+        header=True,
+        index=False,
+        sep='\t',
+    )
+
+
+def export_sequences(database:Sqlite3Database, table_name:str, output_file:str):
+    """Export sequences from a SQLite3 database."""
+
+    query = f"SELECT * FROM {sanitize_identifier(table_name)}"
+
+    database.query_to_fasta(
+        query=query,
+        path=output_file,
+        id_name=DEFAULT_SEQUENCES_ID_NAME,
+        sequence_name=DEFAULT_SEQUENCES_SEQ_NAME,
```

Comment:
This started in `class Sequences` since I thought `id_column` doesn't make sense for FASTA format. It spread from there.

Now, I'm re-thinking the class organization of `StrainBasedFile` → `Seqeunces` | `Metadata`. It's a bit complex and was primarily designed for convenience of [sharing](https://github.com/nextstrain/augur/blob/aba56fe1c4811aa8fbde59e129776138af646c80/augur/db/import_/sqlite3.py#L31-L34) the column names between the different tables, but that won't be necessary if we stick to one table based on separate [discussion](https://github.com/nextstrain/augur/pull/1094#discussion_r1049006853). In that case, these variables would only be needed for database-specific code, so `column` in variable names would be more appropriate.

Reply:
Renamed in [force-push](https://github.com/nextstrain/augur/compare/aba56fe1c4811aa8fbde59e129776138af646c80..0009d4dc197d0a56cdc1d89f0eefe04401063a46).